### DISCUSSION TITLE
Vsync different from vkQuake?

### DISCUSSION THREAD

**Discussion by markanini** [andrei-drexler/ironwail#92] [General]
My setup is 144Hz Vsync on. Is mouse movement lag supposed to feel different between the two ports? Because I think yours feels faster.

**Comment by andrei-drexler** [comment]
I haven't done anything specifically to minimize mouse lag, just general rendering optimizations, so I wouldn't expect any noticeable difference. Subjectively, though, it does feel a bit faster to me, too (144Hz, mouse polling rate set to 1000Hz).
